dbms/CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ add_headers_and_sources(dbms src/Storages/Page/V2/mvcc)
 add_headers_and_sources(dbms src/Storages/Page/V2/VersionSet)
 add_headers_and_sources(dbms src/Storages/Page/V2/gc)
 add_headers_and_sources(dbms src/WindowFunctions)
+add_headers_and_sources(dbms src/TIDB/Schema)
```

Comment:
TIDB -> TiDB?

Reply:
I think it's not needed

Reply:
Well, "TiDB" is more appropriate than "TIDB" since "Ti" stands for Titanium orz. Please rename the directory to TiDB

Reply:
done